scripts/check_max_source_lines.py

```
@@ -0,0 +1,87 @@
+    if not path.is_file():
+        return False
+    if path.suffix not in SOURCE_SUFFIXES:
+        return False
+    if any(part in EXCLUDED_PARTS for part in path.parts):
+        return False
+    return path.as_posix() not in EXCLUDED_FILES
+
+
+def tracked_files(repo_root: Path) -> list[Path]:
+    proc = subprocess.run(
+        ["git", "ls-files"],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)